core/src/banking_stage/transaction_scheduler/receive_and_buffer.rs

```diff
@@ -277,6 +294,244 @@ impl SanitizedTransactionReceiveAndBuffer {
     }
 }
 
+pub(crate) struct TransactionViewReceiveAndBuffer {
+    pub receiver: BankingPacketReceiver,
+    pub bank_forks: Arc<RwLock<BankForks>>,
+}
+
+impl ReceiveAndBuffer for TransactionViewReceiveAndBuffer {
+    type Transaction = RuntimeTransaction<ResolvedTransactionView<Bytes>>;
+    type Container = TransactionViewStateContainer;
+
+    fn receive_and_buffer_packets(
```

Comment:
NOTE: this does **not** run the `bank.check_transactions` because I prioritized safety around not leaking `Bytes` in the container, which is much trickier if we do batching.
We should probably follow this up with a call to clean immediately after, so that we check age on transactions we actually inserted, which is likely better anyway. However, I'd like to profile with and without that change first.